apps/portals/elportal/src/pages/HomePageV2.tsx

```diff
@@ -19,8 +20,27 @@ import {
   topPublicationsSql,
   whatWeDoSql,
 } from '../config/resources'
+import { Typography } from '@mui/material'
+import analyzetheclouds from '@sage-bionetworks/synapse-portal-framework/components/elportal/assets/analyzetheclouds.png'
+import computationaltools from '@sage-bionetworks/synapse-portal-framework/components/elportal/assets/computationaltools.png'
 
 export default function HomePage() {
+  const styledPortalFeatureHighlightsSummaryText = (
+    <>
+      The portal supports secure access to leading cloud-based analysis
+      platforms, such as{' '}
+      <Typography
+        component="span"
+        sx={{ fontWeight: 'bold', color: 'primary.main' }}
+      >
+        CAVATICA
+      </Typography>
```

Comment:
Is this supposed to be a link?

Reply:
Confirmed with Adam this should link to the Cavatica website